ddtrace/internal/datadog/profiling/cmake/AnalysisFunc.cmake

```diff
@@ -1,6 +1,12 @@
 include(CheckIPOSupported)
 
 function(add_ddup_config target)
+    # Profiling native extensions are built with C++17, even though underlying
+    # repo adheres to the manylinux 2014 standard. This isn't currently a
+    # problem, but if it becomes one, we may have to structure the library
+    # differently.
+    target_compile_features(${target} PUBLIC cxx_std_17)
```

Comment:
I'm trying to change the native build in a way that dd_wrapper is built once and shared by ddup, crashtracker and stack_v2. So, I thought it would be natural to put this here as all targets are compiled with c++17.

Reply:
We're building dd_wrapper each time ddup/crashtracker/stack_v2 is built. Building dd_wrapper only once will decrease overall build time for dd-trace-py, and the time to run build_standalone.sh.